src/panels/config/dashboard/ha-config-navigation.ts

```diff
@@ -24,10 +24,13 @@ class HaConfigNavigation extends LitElement {
           ? html`
               <a href=${page.path} aria-role="option" tabindex="-1">
                 <paper-icon-item>
-                  <ha-svg-icon
-                    .path=${page.iconPath}
+                  <div
```

Comment:
The circle containers are not completely square

![image](https://user-images.githubusercontent.com/1444314/142692548-89966559-d89e-4da6-a76f-10521ba51ca5.png)